core/modules/user/user.theme.inc

```diff
@@ -172,15 +172,18 @@ function theme_user_permission_description($variables) {
 function theme_user_admin_roles($variables) {
   $form = $variables['form'];
 
-  $header = array(t('Name'), t('Weight'), t('Operations'));
+  $header = array(t('Name'), t('Status'), t('Weight'), t('Operations'));
```

Comment:
Is this a "status" or a "type"? 🤔

Reply:
I believe 'status' (like nodes can be published or unpublished, roles can be required or not required). Whether or not a role is required isn't defined by the 'type' of role. Unless we use the term 'type' here, and *also* change the 'Required' text to be 'Default' instead. *Then* it would make more sense to use 'type', but I think that's best left for a separate issue (which you can open if you feel so inclined @klonos ).